### PR TITLE
Testing To Do list: part 1

### PR DESCRIPTION
**Pull request by uzairali19**
# Testing Todo-List
This branch is to tests the to-do list's basic functionality and overall functions of the application. It will be carrying out:
- Mock a storage object to "imitate" localStorage operations.
- Mock HTML to test if add/delete functions add or remove exactly one <li> element to/from the list in the DOM.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,73 @@
...
+For Linters:
+
+```bash
+# To check for CSS errors
+npx stylelint "**/*.{css,scss}"
+
+# To check for Javascript errors
+npx eslint .
+```
+
```
Comment (Dipeshtwis) [inline README.md:47]: - [ ] I would recommend mentioning how to test the app. [ HInt: npm test]

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```
Comment (Dipeshtwis) [inline .vscode/settings.json:3]: - [ ] I would recommend removing the unnecessary files from Github to make the project structure good. i.e. `.vscode`, you can put these files or folders to `.gitignore` as well.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @uzairali19 and @GintsM, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: All tests are passing.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .vscode/settings.json (hunk 2 above)
Comment (rloterh) [reply .vscode/settings.json]: This issue is well resolved! ✔️

File: README.md (hunk 1 above)
Comment (rloterh) [reply README.md]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Uzair, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:.
Congratulations! 🎉🥂


<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**